### PR TITLE
cpr 1.9.1

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
Vendors its own `curl`. It shouldn't do that.

File: Formula/cpr.rb [hunk 1]
```diff
@@ -16,14 +16,15 @@ class Cpr < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   uses_from_macos "curl"
 
   def install
     args = std_cmake_args + %w[
```
Comment (carlocab) [inline Formula/cpr.rb:24]:
```suggestion
    args = std_cmake_args + %W[
```
You need `%W` for interpolation.

**Comment by alebcay** [comment]
Note: it looks like compilation works without `openssl@3` on macOS since it will try to link to DarwinSSL (provided by macOS Security framework) instead. We can make the `openssl@3` a Linux-only dependency if desired, but for now have simply added the CMake flag to use `openssl@3` unconditionally across the board.

**Comment by carlocab** [comment]
I'm not sure I get how using macOS `libcurl` with our OpenSSL 3 works, though. I'm surprised it works at all.

**Comment by alebcay** [comment]
Superseded by #110028.

**Pull request closed without merging**